src/style.css

```diff
@@ -1 +1,13 @@
 /* styles go here */
+.fieldBlock {
+  margin-bottom: 20px;
+}
+
+.formField {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.last {
+  margin-bottom: 0;
+}
```

Comment:
there's `:last-child` thing - think of how you can use that instead of styling directly by class `last`